test/cli.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env node, mocha */
+'use strict';
+
+var assert = require('assert');
+var path = require('path');
+var spawn = require('cross-spawn');
+var cli = path.join(__dirname, '../bin/scss-to-json');
+
+function callAndExpectNoErrors(args, cb) {
+  var bin = spawn(cli, args);
+  bin.stdout.setEncoding('utf8');
+  bin.stderr.setEncoding('utf8');
+
+  bin.stderr.once('data', function(data) {
+    throw new Error(data);
+  });
+
+  bin.stdout.once('data', cb);
+}
+
+describe('cli', function() {
```

Comment:
You have CLI flags for help, dependencies, version, and scope. Can you make sure the different cases in your CLI are covered by tests?